### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +27,17 @@ all current channels.
 
 ### Current: 1 day max response time. Slack and Twitter is real-time on workshop days
 
+- [meetup discussion board](https://www.meetup.com/nodeschool-vancouver/discussions/): Public group discussion
```
Comment (keywordnew) [inline README.md:30]: Thanks for adding this.

File: README.md [hunk 2]
```diff
@@ -27,16 +27,17 @@ all current channels.
...
-- [NodeSchool](https://community.vancouvertech.com/c/events/nodeschool) at [YVR Tech Community](https://community.vancouvertech.com): for planning
 
 ### Phased out
 
+- ~- [slack](https://yvrdev.slack.com): for general chat, planning meetups, and
+questions about workshops if you're challenging yourself at home. Go to YVRDev slack
+(sign up [here](https://yvrdev.herokuapp.com/) if you don't have it) and join
+the #nodeschool-yvr channel~
+- ~- [NodeSchool](https://community.vancouvertech.com/c/events/nodeschool) at [YVR Tech Community](https://community.vancouvertech.com): for planning~
 - ~~[meetup.com](https://www.meetup.com/nodeschool-vancouver): general info~~
```
Comment (keywordnew) [inline README.md:41]: Looks like it isn't part of the PR but this may be confusing since we do list meetup.com in another form earlier.

**Review by keywordnew — APPROVED** [review]
Thank you, LGTM. Non-blocking comment left.

File: README.md (hunk 2 above)
Comment (halkeye) [reply README.md:41]: Oh I didn't even notice that was there. I just wanted to get rid of slack.

Suggestions?

Comment (halkeye) [reply README.md:41]: I'm going to merge, we can fix it later

**Pull request merged**